shell.nix

```diff
@@ -10,9 +10,13 @@ let inherit (nixroot) stdenv pkgs lib
     clang-tools
     cmake
     fetchurl
-    nix-gitignore
+    ffmpeg
     gdb
-    python3;
+    libmodplug
+    mp4v2
+    nix-gitignore
+    python3
```

Comment:
Is Python 3 still needed for the CMake build, or should this be kept until SCons has been removed?

Reply:
I can make a separate PR where I remove everything that I think is not needed.

Reply:
Oh, sorry, this actually didn't change. Then just leave it as is.